extensions/common/base64.h

```diff
@@ -139,6 +145,48 @@ inline void encodeLast(uint64_t pos, uint8_t last_char, std::string& ret,
   }
 }
 
+std::string Base64UrlDecode(std::string input) {
```

Comment:
Is this new code copied from somewhere?

Reply:
Yes. This is originated from https://github.com/istio/proxy/blob/3aaff15a85a5c0cc5c23635d4b858eee001df836/src/envoy/http/jwt_auth/jwt.cc#L100.